pkgs/servers/http/tengine/default.nix

```diff
@@ -0,0 +1,82 @@
+{ stdenv, fetchurl, openssl, zlib, pcre, libxml2, libxslt
+, gd, geoip
+, withDebug ? false
+, withMail ? false
+, withIPv6 ? false
```

Comment:
Why disable IPv6? I believe that any server application should support IPv6 by default.

Reply:
Ok, fixed.